backend/macros/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+use proc_macro::TokenStream as TokenStream1;
```

Comment:
Why this weird name? :thinking:

Reply:
Indeed, adding something like this to the config macro invocation:

```rust
some_thing: {
    /// A foo
    foo: u16 = 42,
},
```

generates two warnings:

```
warning: type `Some_thing` should have an upper camel case name
  --> server/src/config.rs:23:5
   |
23 |     some_thing: {
   |     ^^^^^^^^^^ help: convert the identifier to upper camel case: `SomeThing`
   |
   = note: `#[warn(non_camel_case_types)]` on by default

warning: type `Some_thing` should have an upper camel case name
  --> server/src/config.rs:23:5
   |
23 |     some_thing: {
   |     ^^^^^^^^^^ help: convert the identifier to upper camel case: `SomeThing`

warning: 2 warnings emitted
```

Reply:
> Why this weird name? :thinking: 

There is the built-in `proc_macro` library and an external `proc_macro2` one. The latter has exactly the same API but works in more situations... there are reasons to use `proc_macro2`. Everyone in the ecosystem does it. So most of the code just works with `proc_macro2`. Only for the actual entry-point you need the original type from `proc_macro`. And to distinguish its `TokenStream` from the `proc_macro2` one, it is renamed to `TokenStream1`.

---

Your second comment was probably supposed to be on the inline review comment below, right?

Reply:
> Your second comment was probably supposed to be on the inline review comment below, right?

Yes. :sweat_smile:

Reply:
And re: the name: Yeah okay, I understand. I still don't like the name, but I don't care enough. xP